apps/backend/message/Dockerfile

```diff
@@ -1,12 +1,32 @@
 FROM node:20
 
-WORKDIR /app
+RUN apt-get update && apt-get install -y \
+    libgtk2.0-0 \
+    libgtk-3-0 \
+    libgbm-dev \
+    libnotify-dev \
+    libgconf-2-4 \
+    libnss3 \
+    libxss1 \
+    libasound2 \
+    libxtst6 \
+    xauth \
+    xvfb
+
+WORKDIR /app/lib
+COPY --from=lib package*.json ./
+RUN npm install
```

Comment:
This seems to be causing the E2E test failing. Not sure if lib is defined in docker compose